templates/media-add-list.html.twig

```diff
@@ -17,12 +17,16 @@
  */
 #}
 {% if bundles is not empty %}
-    <dl>
+    <ul class="admin-list">
```

Comment:
Infact in administration themes (like seven). The rendering of this structure is really very BAD. WHY? Seven For example rewritten entirely preprocess & template  "node-add-list" for this structure, which gives a really made sexy ! At places to ask themes (admin / front) to rewrite our template each time, I think it would be better that the list is properly formatting.